### PR TITLE
Remove now obsolete suppress lint annotations

### PR DESCRIPTION
**Pull request by ebraminio**
They are no longer needed as the update in Simple-Common project.

Follow up to https://github.com/SimpleMobileTools/Simple-Commons/pull/1230

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt [hunk 1]
```diff
@@ -238,7 +239,7 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
             R.id.menu_force_landscape -> toggleOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
             R.id.menu_default_orientation -> toggleOrientation(ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED)
             R.id.menu_save_as -> saveImageAs()
-            R.id.menu_create_shortcut -> createShortcut()
+            R.id.menu_create_shortcut -> if (isOreoPlus()) createShortcut()
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt:242]: keep the checks in the code please for simplicity, the logic is never here at onOptionsItemSelected

Comment (ebraminio) [reply app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt]: Done https://github.com/SimpleMobileTools/Simple-Gallery/pull/2291/commits/0e1a1cfb90acd875cdbc9a9046156a677ba96cd6

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt [hunk 2]
```diff
@@ -716,8 +715,8 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
         }
     }
 
-    @RequiresApi(Build.VERSION_CODES.O)
     private fun createShortcut() {
+        if (!isOreoPlus()) return
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt:719]: just split it into 3 lines and add brackets, keep the code readable
if (!isOreoPlus()) {
    return
}

Comment (ebraminio) [reply app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt]: Done https://github.com/SimpleMobileTools/Simple-Gallery/pull/2291/commits/dd8cd61450f5dc8c146fbc8aad021c8cc308013d

**Pull request merged**